.github/workflows/performance-check.yml

```
@@ -0,0 +1,118 @@
+          path: ./elementor
+
+      - name: Setup wp-env
+        uses: elementor/elementor-editor-github-actions/actions/setup-wp-env@main
+        with:
+          php: 7.4
+          active-theme: 'hello-elementor'
+          plugins: |-
+            ./elementor
+          themes: |-
+            https://downloads.wordpress.org/theme/hello-elementor.zip
+          mappings: |-
+            templates:./tests/lighthouse/compare-templates
+
+      - name: Setup Elementor ENV
+        uses: elementor/elementor-editor-github-actions/actions/setup-elementor-env@main
+        with:
+          env: 'testing'
+          enable-svg-upload: true
+          experiments: |-
+            e_optimized_markup:true
+            editor_v2:true
+            atomic_widgets:true
+            global_classes:true
+          templates: |-
+            templates
+
+      - name: Warm up the cache
+        run: |
+          curl -o /dev/null http://localhost:8889/v3-only/ &&
+          curl -o /dev/null http://localhost:8889/mixed-template/
+
+      - name: Run Performance Check
+        uses: elementor/elementor-editor-github-actions/actions/run-lighthouse-tests@main
+        id: run-lighthouse-tests
+        with:
+          number-of-runs: 3
+          urls: |-
+            v3:http://localhost:8889/v3-only/
+            mixed:http://localhost:8889/mixed-template/
+          categories: |-
+            performance
+
+      - name: Check if score meets the threshold
+        uses: actions/github-script@v7
+        env:
+          V3_SCORE: ${{ steps.run-lighthouse-tests.outputs.v3-performance-median-score }}
+          MIXED_SCORE: ${{ steps.run-lighthouse-tests.outputs.mixed-performance-median-score }}
+        with:
+          script: |
+            const v3Score = parseFloat( process.env.V3_SCORE );
+            const mixedScore = parseFloat( process.env.MIXED_SCORE );
+            const threshold = parseFloat( process.env.PERFORMANCE_SCORE_THRESHOLD );
+            
+            if ( mixedScore < v3Score ) {
+               core.setFailed( `The performance score of the mixed page (v3 and v4 elements) is ${mixedScore}, which is below the performance score of the V3 page ${v3Score}.` );
+            }
+            
+            if ( mixedScore < threshold ) {
+               core.setFailed( `The performance score of the mixed page (v3 and v4 elements) is ${mixedScore}, which is below the threshold of ${threshold}.` );
+            }
+
+      - name: Upload reports
+        id: upload-reports
+        if: steps.run-lighthouse-tests.outputs.reports-path
+        uses: actions/upload-artifact@v4
+        with:
+          name: compare-v4-to-v3
+          path: ${{ steps.run-lighthouse-tests.outputs.reports-path }}
+          retention-days: 1
+
+      - name: Send Slack notification
+        uses: slackapi/slack-github-action@v2.0.0
+        env:
+          RUN_ID: ${{ github.run_id }}
+          ARTIFACT_ID: ${{ needs.upload-reports.outputs.artifact_id }}
```

```suggestion
          ARTIFACT_ID: ${{ steps.upload-reports.outputs.artifact_id }}
```

(maybe put the download URL instead?)

---

Done, and I put the ID and not the URL for security reasons